gen3-integration-tests/services/indexd.py

```diff
@@ -92,6 +124,22 @@ def delete_files(self, guids: list, user="indexing_account"):
             except Exception as e:
                 logger.exception(msg=f"Failed to delete record with guid {guid} : {e}")
 
+    def delete_file_indices(self, records: dict):
+        for key, val in records.items():
+            try:
+                indexd_record = self.get_record(indexd_guid=val["did"])
+                indexd_rev = self.get_rev(json_data=indexd_record)
+                if indexd_rev is None:
+                    logger.info("Indexd record returned None")
+                    continue
+                logger.info(f"{val['did']} found, performing delete.")
+                self.delete_record(guid=indexd_record["did"], rev=indexd_rev)
+            except Exception as e:
+                if "404" not in f"{e}" and "did" not in f"{e}":
+                    logger.error(f"404 status code not returned. Exception : {e}")
+                    raise
```

Comment:
the error message is not clear - the problem is not that we didn't get a 404, it's that something went wrong when trying to delete the file

instead of logging the error and then raising an exception with no details, i'd also suggest doing this:
```
raise Exception("err message here")
```